gemini.ts

```diff
@@ -32,6 +33,63 @@ export const extractQuestions = async (text: string) => {
   }
 }
 
+export const getSystemPrompt = async (
+  question: string,
+  type: QuestionnaireType,
+  customerType: CustomerType
+) => {
+  const [info, policies, methodology, similarAnswers] = await Promise.all([
+    readFile(path.join('system-prompt-files', 'info.txt'), 'utf-8'),
+    readFile(path.join('system-prompt-files', 'policies.txt'), 'utf-8'),
+    readFile(path.join('system-prompt-files', 'methodology.txt'), 'utf-8'),
+    getSimilarAnswers([question])
+  ])
+
+  let similarAnswer = undefined
+  if (similarAnswers.length && similarAnswers?.[0]?.neighbors?.length) {
+    similarAnswer = similarAnswers[0].neighbors[0].answer
+    console.log('similarAnswer', similarAnswer)
+  }
+
+  return `
+      You are an expert at answering RFI and security questions for Scope3. 
+      IMPORTANT: Your response must only be the answer to the question. Do not include any additional text or markdown formatting. Keep the answer concise and to the point.
+      Note that these questions are all regarding Scope3, and may not always be phrased as a question.
+      ${
+        // Only add this instruction if the questionnaire type is not OTHER
+        type !== QuestionnaireType.OTHER
+          ? `This is a ${type} questionnaire for a potential ${customerType} customer. Use this information to better answer the questions.`
+          : ''
+      }
+      ${
+        similarAnswer
```

Comment:
Should we check the distance and only include if it's like `> 0.5` or something? I'm worried that if we get something unlike anything we've ever seen, this may actually make the answer worse.

Reply:
oh yes yes

Reply:
done, I set it to 0.75